### PR TITLE
Replace %-formatting by f-strings in code examples

### PR DESCRIPTION
**Pull request by konrad**
#### What does this PR implement/fix? Explain your changes.

Translates the old-fashined %-formatting to modern f-strings.

#### How should this PR be tested?

Run examples.

#### Any other comments?

It is not clear at which point changes have to be mate to the chancelog in this case.

### CODE REVIEW

File: examples/30_extended/tasks_tutorial.py [hunk 1]
```diff
@@ -75,15 +75,15 @@
...
+print("First 5 of {len(tasks)} tasks:")
 print(tasks.head())
 
 ############################################################################
 # Furthermore, we can list tasks based on the dataset id:
 
 tasks = openml.tasks.list_tasks(data_id=1471)
 tasks = pd.DataFrame.from_dict(tasks, orient='index')
-print("First 5 of %s tasks:" % len(tasks))
+print("First 5 of {len(tasks)} tasks:")
```
Comment (mfeurer) [inline examples/30_extended/tasks_tutorial.py]: Isn't there an `f` missing in front of the string?

File: examples/30_extended/tasks_tutorial.py [hunk 2]
```diff
@@ -75,15 +75,15 @@
 
 tasks = openml.tasks.list_tasks(tag='OpenML100')
 tasks = pd.DataFrame.from_dict(tasks, orient='index')
-print("First 5 of %s tasks:" % len(tasks))
+print("First 5 of {len(tasks)} tasks:")
```
Comment (mfeurer) [inline examples/30_extended/tasks_tutorial.py]: Isn't there an `f` missing in front of the string?

File: examples/30_extended/tasks_tutorial.py (hunk 1 above)
Comment (konrad) [reply examples/30_extended/tasks_tutorial.py]: Right!

File: examples/30_extended/tasks_tutorial.py (hunk 2 above)
Comment (konrad) [reply examples/30_extended/tasks_tutorial.py]: Also right!

**Comment by mfeurer** [comment]
Thank you very much for your contribution! Further contributions are very welcome ;)

**Comment by konrad** [comment]
Thanks, for the motivating feedback, @mfeurer!

**Pull request merged**